descriptors/ModuleDescriptor-template.json

```diff
@@ -1928,6 +1928,38 @@
       ],
       "visible": false
     },
+    {
+      "permissionName": "modperms.circulation.requests.item.delete",
+      "displayName": "module permissions for one op",
+      "description": "to reduce X-Okapi-Token size",
+      "subPermissions": [
+        "circulation-storage.requests.item.get",
+        "circulation-storage.requests.collection.get",
+        "circulation-storage.loans.collection.get",
+        "circulation-storage.loans.item.get",
+        "inventory-storage.items.collection.get",
+        "inventory-storage.items.item.get",
+        "inventory-storage.holdings.collection.get",
+        "inventory-storage.holdings.item.get",
+        "inventory-storage.instances.collection.get",
+        "inventory-storage.instances.item.get",
+        "inventory-storage.locations.collection.get",
+        "inventory-storage.locations.item.get",
+        "inventory-storage.location-units.institutions.item.get",
+        "inventory-storage.location-units.campuses.item.get",
+        "inventory-storage.location-units.libraries.item.get",
+        "inventory-storage.service-points.collection.get",
+        "inventory-storage.service-points.item.get",
+        "users.item.get",
+        "users.collection.get",
+        "usergroups.collection.get",
+        "usergroups.item.get",
+        "inventory-storage.location-units.libraries.collection.get",
+        "circulation-storage.requests.item.delete",
+        "circulation-storage.request-batch.item.post"
+      ],
+      "visible": false
+    },
```

Comment:
I believe that this fake permission set workaround is no longer needed since changes were made by the Core Platform team to generate these on the fly when a module is enabled for a tenant.

@craigmcnally Please could you confirm this?

Reply:
Do you mean that we can just add all the permissions to endpoint's `modulePermissions`

Reply:
> Do you mean that we can just add all the permissions to endpoint's `modulePermissions`?

Yes, I believe so. I don't want to do that until this is confirmed by @craigmcnally or another member of the Core Platform team.

Reply:
@bohdan-suprun Given we haven't had an answer from Craig, I'll approve this as is, and we can clean this permission set up with the others